static/quickstart.js

```diff
@@ -4,8 +4,9 @@
   var outputVolumeBar = document.getElementById('output-volume');
   var inputVolumeBar = document.getElementById('input-volume');
   var volumeIndicators = document.getElementById('volume-indicators');
+  var device, outgoingCall;
```

Comment:
`outgoingCall` doesn't seem to be used unless that's being hidden in the diff that I'm looking at, and is redeclared on line 60. Maybe it isn't needed here?
```suggestion
  const volumeIndicators = document.getElementById('volume-indicators');<br class="Apple-interchange-newline">
  let device;
```

Side note: maybe it's outside the scope of this change, but I don't think we should demonstrate using `var` in our JS code samples since it's function scoped and generally unwieldy. Most of these variables can be declared using `const` instead, and a small number like `device` can be declared with `let` since they're mutable 😄 

Happy to bash on this a bit if that'd help~